response_operations_ui/templates/sign_in.html

```diff
@@ -42,46 +53,9 @@
     {% endwith %}
     <form method="post" class="form">
         {{ form.csrf_token }}
-        {% if form.errors %}
-            {% set errorData = [] %}
-            {% call
-                onsPanel({
-                    title: "This page has 1 error" if form.errors|length == 1 else "There are {{ form.errors|length }} errors on this page",
-                    type: "error"
-                })
-            %}
-                <p>These <strong>must be corrected</strong> to continue.</p>
-                {% for error in form.username.errors %}
-                    {% do errorData.append(
-                        {
-                            "text": error,
-                            "index": true,
-                            "url": "#username",
-                            "classes": "ons-js-inpagelink"
-                        }
-                    ) %}
-                {% endfor %}
-                {% for error in form.password.errors %}
-                    {% do errorData.append(
-                        {
-                            "text": error,
-                            "index": true,
-                            "url": "#password",
-                            "classes": "ons-js-inpagelink"
-                        }
-                    ) %}
-                {% endfor %}
-                {{
-                    onsList({
-                        "classes": "list--bare",
-                        "itemsList": errorData
-                    })
-                }}
-            {% endcall %}
-        {% endif %}
         <h2>Sign in</h2>
         {% if failed_authentication or form.username.errors | length > 0 %}
-            {% set error = { "text": "Email address is required" } %}
+            {% set errorUN = { "text": "Email Address is required", "id": 'username_error' } %}
```

Comment:
At first glance this variable confused me, maybe something like `error_username`?